src/flamegraph.cpp

```diff
@@ -680,6 +799,23 @@ void FlameGraph::saveSvg(const QString& fileName) const
     m_rootItem->setBrush(oldBrush);
 }
 
+void FlameGraph::setColorScheme(ColorScheme _scheme)
+{
+    scheme = _scheme;
+
+    std::function<void(FrameGraphicsItem*)> traverseFunction = [&traverseFunction](FrameGraphicsItem* item) {
+        item->setBrush(brush(item->symbol(), BrushType::Hot));
+        for (const auto& child : item->childItems()) {
+            traverseFunction(reinterpret_cast<FrameGraphicsItem*>(child));
```

Comment:
why reinterpret_cast? very bad idea - use `static_cast` instead and make sure you get the `const` correct above